src/components/Share/modal/index.js

```diff
@@ -300,6 +310,9 @@ export function ShareModal({ isOpen, onClose, postUrl }) {
                                             onClick={() => {
                                                 sendDM(selectedUsers, shareUrl);
                                                 removeSelectedUsers();
+                                                showSuccessMessageWithTimeout(
```

Comment:
We should add a check here. Currently if a user is not logged in, this message would still be displayed.
Expected behaviour for logged out user is:
- Can click on share button and search users.
- Can select any number of users but as user hits the send button, the alert should appear asking him to sign in and the notification shouldn't be visible unless the meme is sent.

Reply:
Completed this change and sent a pull request